apps/elewa-website/src/app/app.component.ts

```diff
@@ -7,4 +8,22 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'elewa-website';
+
+  textContents: ContentText[] = [
+    {
+      title: 'From here to the world',
```

Comment:
No need to submit the data. This will be provided by a component that resuses the one you created